packages/react/src/components/MapCard/ScrollingControls.jsx

```diff
@@ -59,7 +59,7 @@ const ScrollingControls = ({
   return (
     <div data-testid={testID} className={classnames(`${BASE_CLASS_NAME}__container`)}>
       <Button
-        data-testid={`${testID}-scroll-up`}
+        testId={`${testID}-scroll-up`}
```

Comment:
Tested it and `data-testid={`${testID}-scroll-up`}` doesn't work so this is overwriting the button default testId which is "Button"...